src/server/getters/database.ts

```diff
@@ -51,14 +51,14 @@ async function queryModifierUserland<T>(
   defaultSortOrder: string,
   sortLimitParams: Partial<SortLimit>,
 ): Promise<Array<T>> {
-  type RowWithSort = T&{ xMySorterFieldx: BigNumber };
+  type RowWithSort = T & { xMySorterFieldx: BigNumber };
```

Comment:
A bunch of the whitespace changes are from tsformat auto-formatting files I change.